x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/use_columns.tsx

```diff
@@ -213,7 +213,7 @@ export const useColumns = (
       'data-test-subj': 'transformListColumnDescription',
       name: i18n.translate('xpack.transform.description', { defaultMessage: 'Description' }),
       sortable: true,
-      truncateText: true,
+      truncateText: false,
```

Comment:
This now disables truncation without any limit whatsoever which we'd like to avoid. An alternative would be to make it work with multi-line-truncation. You could copy the approach taken in here to truncate for example after 3 lines: https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/common/truncated_text.tsx